app/src/ui/tab-bar-item.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from 'react'
```

Comment:
This file and others like is are here because of the `no-use-before-declare` rule being unhappy with our previous state of affairs.